conf/viewvc.conf.dist

```diff
@@ -360,6 +360,28 @@
 ##---------------------------------------------------------------------------
 [options]
 
+## repos_locale: (Unix/Linux only) (this comment is tentative)
+## Specifies LC_CTYPE locale for repository paths on your server.
+## Repository paths specified in cvs_roots and/or svn_roots can be
+## non-ascii paths represented in Unicode.  We assume the viewvc.conf
+## file content is encoded by the locale that specified by locale
+## environment variables when ViewVC starts.  However, repository paths
+## can be placed in the path encoded by some encoding other than
+## ViewVC's start up locale.  So it can be override this value per
+## repos root. This value should be valid locale that supported by
+## your system environment.  If this is not set, ViewVC uses the locale
+## on strarting.
+##
+## (Warning) If setlocale is not thread safe on your system and if
+## you use mod_wsgi in daemon mode, do not set this value.
+## Otherwise, you might experience unexpected crashes or errors.
+##
+## Examples:
+## repos_locale = en_US.UTF-8
+## repos_locale = Russian_Russia.1251
+##
+#repos_locale =
+
```

Comment:
Thank you for the review.  It makes sense to use "root", it's important to keep consistency of terms.

I forgot to mention that its locale also affects iterpretation of internal paths in CVS repositories, because those paths are just relative paths from root path in file system.

So I'd like to add "This is also used for showing CVS's internal paths as string of the locale." before "If set,  this value must ...".

>  Hopefully I actually understand the problem you're solving!

Yes, I think you do.  With this PR,  ViewVC can even show root listing roots which have paths of different encodings if those have correct locale settings in per-root overriding options.